### PR TITLE
closes #1721

### LINKED ISSUE #1721: fix: profile-addition spelling
### Description

The "chore: profile-addition" label is misspelt.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by RealSollyM**
fixed the spelling of the label to match the pre-existing label.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1722"></a>

### CODE REVIEW

File: reviewpad.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ workflows:
     if:
       - rule: $hasFilePattern("public/data/**")
     then:
-      - '$addLabel("chore: profile-adition")'
+      - '$addLabel("chore: profile-addition")'
```
Comment (Panquesito7) [inline reviewpad.yml:64]: It'd be best if we could use `✍ chore: profile addition`; not sure if that's possible, though.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (RealSollyM) [reply reviewpad.yml]: I have now added the icon too.

**Review by Panquesito7 — APPROVED** [review]
Awesome! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Review by vedanshi555 — APPROVED** [review]

File: reviewpad.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ workflows:
     if:
       - rule: $hasFilePattern("public/data/**")
     then:
-      - '$addLabel("chore: profile-adition")'
+      - '$addLabel("✍ chore: profile-addition")'
```
Comment (Pradumnasaraf) [inline reviewpad.yml:64]:
```suggestion
      - '$addLabel("✍ chore: profile addition")'
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Panquesito7 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

**Pull request merged**